tests/integration/test_forest_plot_lib.py

```
@@ -0,0 +1,98 @@
+        
+        # Should still work, just not display interaction p-values
+        fig = create_forest_plot(df_no_int, 'Est', 'Lower', 'Upper', 'Subgroup')
+        
+        assert fig is not None
+
+    def test_empty_dataframe(self):
+        """🚫 Test handling of empty input"""
+        df_empty = pd.DataFrame(columns=['Subgroup', 'Level', 'Est', 'Lower', 'Upper'])
+        
+        # Verify that empty DataFrame raises ValueError
+        with pytest.raises(ValueError, match="empty|no data"):
```

Ruff (0.14.10)

Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)